BookingsApi/BookingsApi.AcceptanceTests/BookingsApi.AcceptanceTests.csproj

```diff
@@ -15,18 +15,20 @@
     <PackageReference Include="FluentAssertions" Version="5.10.3" />
     <PackageReference Include="Microsoft.Extensions.Configuration.UserSecrets" Version="5.0.0" />
     <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
-    <PackageReference Include="nunit" Version="3.13.0" />
+    <PackageReference Include="nunit" Version="3.13.1" />
```

Comment:
VH.AcceptanceTests.Common had an old reference and looks like I had to update it to the latest as one of the tests was failing (test name: Should_remove_participants). However that package update required all these package references to be updated (nunit, SpecFlow, SpecFlow.NUnit, SpecFlow.Tools.MsBuild.Generation,VH.AcceptanceTests.Common) both in acceptance test project and in integration test project. It might be worth reviewing whether we need these hardcode references here at all